antora-ui-camel/src/css/footer.css

```diff
@@ -52,6 +52,12 @@ footer .footer dl dt {
   margin-bottom: 0.5rem;
 }
 
+.stroke {
```

Comment:
Can we scope this to footer?

```suggestion
footer .footer .stroke {
```

Reply:
ok